homeassistant/components/influxdb/const.py

```diff
@@ -72,7 +72,7 @@
 EVENT_NEW_STATE = "new_state"
 DOMAIN = "influxdb"
 API_VERSION_2 = "2"
-TIMEOUT = 5
+TIMEOUT = 500
```

Comment:
8 minutes? Isn't that a big high?

It defaults to 10 seconds from the upstream API library for Influx V2.
Setting this to 30, maybe at max a minute.

Reply:
Care to define/share reasoning?

Reply:
I honestly just added a `0` to make it work and 50 didn't work on my machine or at least errored that time. I might have messed up something else while trying though. If you prefer 30 I can try that again.

Reply:
Also, the golang package (which I used to carry out tests outside of home-assistant) has 60 seconds in the example in the README https://github.com/influxdata/influxdb-client-go

Reply:
See below. 30 is still to low